firedrake/preconditioners/assembled.py

```diff
@@ -107,12 +121,12 @@ def view(self, pc, viewer=None):
             self.pc.view(viewer)
 
 
-class ExplicitSchurPC(AssembledPC):
+class AuxiliaryOperatorPC(AssembledPC):
     """A preconditioner that builds a PC on a specified form.
     Mainly used for describing approximations to Schur complements.
     """
 
-    _prefix = "schur_"
+    _prefix = "aux_"
```

Comment:
Not in this change, but I might make this prefix public, and therefore more obviously overridable, so you can say, for example:

```python
class MassPC(AuxiliaryOperatorPC):
    prefix = "mass_"
```

And use a more "natural" prefix for the configuration.